scripts/eval.py

```diff
@@ -0,0 +1,14 @@
+import insight.database.db_connection as connection
+import insight.database.utils as util
+import insight.metrics as mt
+
+DATA_DIR = "https://raw.githubusercontent.com/synthesized-io/datasets/master"
+DATASET = "tabular/templates/credit.csv"
+URL = f"{DATA_DIR}/{DATASET}"
+
+df = util.get_df(f"{DATA_DIR}/{DATASET}")
```

Comment:
It would be good to use `synthesized_datasets` here:

```python
import synthesized_datasets as datasets

data = datasets.CREDIT.credit
df = data.load()
df.name = data.name
```